ui/job-view/details/tabs/autoclassify/ErrorLine.jsx

```diff
@@ -144,7 +144,7 @@ export default class ErrorLine extends React.Component {
    */
   onOptionChange(option) {
     this.initOption(option);
-    this.setState({ selectedOption: option });
```

Comment:
This was actually an older bug I'd missed.  When setting state, you need to set it to a new ``Object`` not just the same object that had a field modified, as we did in this case.  So doing the spread remedies that.